mls_validation_service/src/config.rs

```diff
@@ -18,4 +18,8 @@ pub(crate) struct Args {
     // A path to a json file in the same format as chain_urls_default.json in the codebase.
     #[arg(long)]
     pub(crate) chain_urls: Option<String>,
+
+    // The size of the cache to use for the smart contract signature verifier.
+    #[arg(long, default_value_t = 20)]
```

Comment:
Probably want this to be more like a few thousand at least, right? How big is a cache item?

Reply:
120B at minimum, if I'm correct:
```
pub struct CacheKey {
    pub address: String,
    pub chain_id: String,
    pub hash: [u8; 32],
    pub signature: Vec<u8>,
    pub block_number: Option<u64>,
}
````

On the other hand, probably it's okay if we just cache hash+signature+block_number.

Reply:
Yeah, you could probably up this to 10k at least

Reply:
Removed the whole logic regarding CacheKey, instead now a CacheKey is just a hash `type CacheKey = [u8; 32];`. 32 bytes gives us enough uniqueness.

Default cache size is 10000.